content/docs/command-reference/update.md

```diff
@@ -82,15 +81,19 @@ stable.
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
 
-## Example: Updating imported artifacts to a specified revision
+## Example: Updating fixed-revision imports to a different version
```

Comment:
`fixed-revision imports` sounds very technical :)

Reply:
Changing to "Updating fixed revisions to a different version".